wrangler-proto/src/main/java/io/cdap/wrangler/proto/workspace/v2/WorkspaceDetail.java

```diff
@@ -21,17 +21,27 @@
 
 import java.util.List;
 import java.util.Objects;
+import javax.annotation.Nullable;
 
 /**
  * Full information about the workspace, including sample data
  */
 public class WorkspaceDetail {
   private final Workspace workspace;
   private final List<Row> sample;
+  @Nullable
+  private final byte[] sampleAsBytes;
```

Comment:
I think we will have to re-generate equals and hashcode methods

Reply:
Done.